building/devices/zynqmp.rst

```diff
@@ -1,150 +1,74 @@
 .. _zynqmp:
 
 #########################
-ZynqMP zcu10x and Ultra96
+Zynq MPSoC
 #########################
-Instructions below show how to run OP-TEE on ZynqMP zcu10x and Ultra96 board.
+Instructions below show how to run OP-TEE on Zynq MPSoC based boards.
 
 Supported boards
-****************
-This makefile supports the following ZynqMP boards:
-	
-	* zcu102
-	* zcu104
-	* zcu106
-	* Ultra96v1
-
-Setting up the toolchain
-************************
-This build chain heavily relies on Petalinux 2018.2 therefore the first step is 
-to download and install the Petalinux 2018.2 toolchain from the Xilinx website 
-(`Downloads`_). Then, you have to download the needed BSP file from the Xilinx 
-website (`Downloads`_). You may have to create a free Xilinx account to proceed 
-with the two previous steps.
-
-Configuring and building for zcu102 board
 *****************************************
-First, create a new directory which will be used as root directory:
-
-.. code-block:: bash
-
-	$ mkdir -p ~/petalinux-optee
-	$ cd ~/petalinux-optee
-
-Then, copy the zcu102 BSP file into the newly created directory:
-
-.. code-block:: bash
-
-	$ cp ~/Downloads/xilinx-zcu102-v2018.2-final.bsp .
-
-Git clone the ``build`` repository of the OP-TEE project and source the 
-Petalinux settings:
-
-.. code-block:: bash
-
-	$ git clone https://github.com/OP-TEE/build
-	$ cd ./build
-	$ source /path/to/petalinux/settings.sh
-
-Finally, use the following commands to create, patch, configure and build the 
-Petalinux project. Petalinux is a powerful but very slow tool, each command may 
-take a while according to the capabilities of your computer.
-
-.. code-block:: bash
-	
-	$ make -f zynqmp.mk
-
-Once the last command ends up you are ready to run QEMU tool or to make a 
-bootable SD card. To run QEMU:
-
-.. code-block:: bash
-
-	$ make -f zynqmp.mk qemu
++--------------+--------------+----------------------+
+| Board Name   | Manufacturer | Hardware Description |
++==============+==============+======================+
+| ZCU102       | Xilinx       | `ZCU102 Website`_    |
++--------------+--------------+----------------------+
```

Comment:
This change removes build instructions for board zcu104, zcu106 and Ultra96v1.
@mimok, @ricardosalveti, are you ok with the proposed change in the documentation?

Reply:
Yeah, please don't remove the build instructions for the other targets, are platform_config.h has the right checks based on the flavor used (e.g. ultra96 uses a different uart base address).

It is fine to have a more generic build instruction for the ZCU102 family, but also leave the part describing for other ZynqMP boards.

Reply:
The instructions for the other targets should be similar, only PLATFORM will be different. Just don't remove the older block that says how to build for the other targets and we should be good.